tgui/packages/tgui/interfaces/PreferencesMenu/antagonists/antagonists/opportunist.ts

```diff
@@ -0,0 +1,16 @@
+import { multiline } from "common/string";
+import { Antagonist, Category } from "../base";
+import { THIEF_MECHANICAL_DESCRIPTION } from "./thief";
+
+const Opportunist: Antagonist = {
+  key: "opportunist",
+  name: "Opportunist",
+  description: [
```

Comment:
Like the other midrounds, you should have an objective line in here explaining this will convert an existing player in the round. Look at sleeper agent for what I mean.

Reply:
Thank you, will add